client/store/order.js

```diff
@@ -1,3 +1,5 @@
+//POSSIBLY USELESS, DELETE??
+
```

Comment:
I made this comment just to indicate that this file may be unused. We don't have to worry about deleting it right now, but should review it before our final submission.